### PR TITLE
d_a_obj_digholl & d_a_obj_kjgjs OK

### PR DESCRIPTION
**Pull request by Trueffeloot**

### CODE REVIEW

File: rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp [hunk 1]
```diff
@@ -1,292 +1,133 @@
...
+        }
+    } else {
+        if ((player->current.pos.abs2XZ(current.pos) < 250000.0f) &&
+            (fabsf(current.pos.y - player->current.pos.y) < 200.0f))
+        {
+            daPy_py_c::setLookPos(&mAttentionInfo.mPosition);
+        }
+    }
+
+    if (sqrtf(PSVECSquareDistance((Vec*)&player->current, (Vec*)&current)) < 1000.0f) {
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp:85]: this should be `player->current.pos.abs(current.pos)` if possible

File: rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp [hunk 2]
```diff
@@ -1,292 +1,133 @@
...
+    }
+    if (i_fopAcM_isSwitch(this, field_0x569)) {
+        field_0x568 = 2;
+        field_0x56b = 1;
+    }
+    fopAcM_SetMin(this, -60.0f, 0.0f, -100.0f);
+    fopAcM_SetMax(this, 60.0f, 10.0f, 100.0f);
+    mAttentionInfo.mPosition = current.pos;
+    mEyePos = mAttentionInfo.mPosition;
+    mAttentionInfo.field_0x4[3] = ' ';
```
Comment (TakaRikka) [inline rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp:33]: this should probably be `mAttentionInfo.field_0x4[3] = 0x20` instead

**Review by TakaRikka — COMMENTED** [review]
couple things, also progress conflict needs to be fixed

File: rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp [hunk 3]
```diff
@@ -1,292 +1,133 @@
...
 /* 80BDC7D8-80BDCAEC 000238 0314+00 1/1 0/0 0/0 .text            execute__14daObjDigholl_cFv */
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-asm void daObjDigholl_c::execute() {
-    nofralloc
-#include "asm/rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl/execute__14daObjDigholl_cFv.s"
+int daObjDigholl_c::execute() {
+    daPy_py_c* player = daPy_getLinkPlayerActorClass();
+    if (dComIfGp_getLinkPlayer()->i_checkNoResetFlg1(daPy_py_c::FLG1_IS_WOLF) &&
```
[the remark this answers is not in the document]
Comment (Trueffeloot) [reply rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp:54]: using the inline generates extra asm instructions and swaps some

[the remark this answers is not in the document]
Comment (Trueffeloot) [reply rel/d/a/obj/d_a_obj_digholl/d_a_obj_digholl.cpp:54]: nvm forget it. Found a matching way using the inline

**Comment by Trueffeloot** [comment]
alright fixed all things 👍

**Review by TakaRikka — APPROVED** [review]

**Pull request merged**